frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardEntityIdCard/DashboardEntityIdCard.module.css

```diff
@@ -0,0 +1,15 @@
+.EntityId {
+  margin-bottom: 0.5rem;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+}
```

Comment:
nit: instead of doing last-child shenanigans, I'd just put them all in a `Stack` with a `gap`

Reply:
Good Call